packages/key-manager/src/agent/SphereonKeyManager.ts

```diff
@@ -58,4 +59,24 @@ export class SphereonKeyManager extends VeramoKeyManager {
   async keyManagerListKeys(): Promise<ManagedKeyInfo[]> {
     return this.localStore.list({}) // FIXME there are no args it seems
   }
+
+  async keyManagerGet({ kid }: IKeyManagerGetArgs): Promise<IKey> {
+    try {
+      const key = await this.localStore.get({ kid })
+      return key
+    } catch (e) {
+      const keys: ManagedKeyInfo[] = await this.keyManagerListKeys()
```

Comment:
Hmzz, please create a ticket that we need to do this in a nicer way in the future.

Reply:
added the ticket and todo: https://sphereon.atlassian.net/browse/SDK-28